oaknational/Oak-Web-Application
# style: update footer from designs

## Description

- Update footer styles from [figma](https://www.figma.com/file/lgns6S3DT4CbCqKQrEf0z4/Oak-Design-System-2022?node-id=137%3A20794)
- adds Oak logo and with text variant
- adds footer pen line SVG
- updates social media icons
- fix line height props

## Issue(s)

Storybook is not showing SVG background images
Storybook is not showing passed in notification prop (Join Beta)

Fixes #358 

## How to test

1. Go to {cloud link} or storybook / headers and footers
2. Click on _______
3. You should see _______



## Checklist
